playwright/e2e/sliding-sync/sliding-sync.spec.ts

```diff
@@ -188,30 +156,6 @@ test.describe("Sliding Sync", () => {
         ).not.toBeAttached();
     });
 
-    test("should not show unread indicators", async ({ page, app, joinedBot: bot, testRoom }) => {
```

Comment:
Can we invert this test rather than removing it?

Reply:
Sure... done.